README.md

```diff
@@ -17,76 +16,119 @@ _A table of contents to help people find things_
+No code will run automatically unless you are using the Build Service. You must either construct an `o-message` object or fire an o.DOMContentLoaded event, which `o-message` listens for.
 
-No code will run automatically unless you are using the Build Service.
-You must either construct an `o-component-boilerplate` object or fire the `o.DOMContentLoaded` event, which oComponent listens for.
 
-#### Constructing an o-component-boilerplate
+#### Constructing an o-message
 
+If you have set up your message declaratively, and are using default o-message classes, use the following:
 ```js
-const oComponentBoilerplate = require('o-component-boilerplate');
-
-const oComponentBoilerplate = new oComponentBoilerplate();
+const oMessage = require('o-message');
+const messageElement = document.getElementById('my-message');
+const importantMessage = new oMessage(messageElement, { content: { highlight: 'Success' } });
 ```
+The second argument that `oMessage` accepts is an (options object)[#options], which can be used to change some behaviour and some markup of a message.
 
-#### Firing an oDomContentLoaded event
+If you're setting up a message without existing DOM elements, oMessage will construct an element for you when it is set up like this:
 
 ```js
-document.addEventListener('DOMContentLoaded', function() {
-	document.dispatchEvent(new CustomEvent('o.DOMContentLoaded'));
+const oMessage = require('o-message');
+const importantMessage = new oMessage(null, {
+	theme: 'error',
+	content: {
+		highlight: 'Something has gone wrong.'
+		detail: 'The quick brown fox did not jump over the lazy dogs.'
+	}
 });
 ```
 
-### Sass
-_Remember to start your codeblocks with three backticks and "sass" so your markup is syntax highlighted correctly._
+#### Options
+`o-message` allows for several configuration options that will change the type of message and its visual styling.
+
+The only required options are listed in the example _above_. These are:
+- `theme`: String. The theme to apply to the message. The available themes are 'success', 'error' and 'neutral', which are designed to be used with corresponding messages.
+- `content`: Object. Holds the following values for text properties:
+	- `highlight`: String. The highlighted text in a message. Defaults to `null`
+	-	`detail`: String. The detail about the nature of a message.
+
+The following options are not required, and all have a default value:
+
+- `autoOpen`: Boolean. Whether to open the message automatically, defaults to `true`.
+- `messageClass`: String. The base class name for the component's elements, defaults to `o-message`.
+- `messageType`: String. The type of message that you want to initialise (e.g. alert, cookie, marketing.) Currently, the only available choice is `'alert'`, which is also the default value.
+- `bleed`: Boolean. Whether the message bleeds across the viewport, defaults to `false`,
```

Comment:
Based on the readme alone I'm not sure how the options `inline` and `bleed` work together. Isn't one option needed? I.e. if `inline` is false then the alert will bleed.

Reply:
I'll update the readme or change the code, but `inline` and `bleed` are different things.
I see your point, so I'll try and make an intelligible change.